src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -96,15 +96,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $env = $this->getContainer()->getParameter('kernel.environment');
+        $address = $input->getArgument('address');
+
+        if (false === strpos($address, ':')) {
+            $output->writeln('The address has to be of the form <comment>bind-address:port</comment>.');
```

Comment:
I don't know if `bind-address` is clear enough for end users. But I don't know either how to better explain that it can be an IP address or a host name.